app/javascript/controllers/evaluator_recusal_modal_controller.js

```diff
@@ -0,0 +1,44 @@
+import ModalController from "./modal_controller"
+
+export default class extends ModalController {
+  static values = {
+    assignmentId: String,
+    submissionId: String,
+    evaluationId: String
+  }
+
+  open(event) {
+    event.preventDefault()
+    this.assignmentIdValue = event.currentTarget.dataset.assignmentId
+    this.submissionIdValue = event.currentTarget.dataset.submissionId
+    this.evaluationIdValue = event.currentTarget.dataset.evaluationId
+    this.modalTarget.showModal()
+  }
+
+  evaluatorRecusal(event) {
+    const recusalPath = this.evaluationIdValue ? 
+      `/evaluations/${this.evaluationIdValue}/recuse` : 
+      `/submissions/${this.submissionIdValue}/evaluations/recuse`
+
+    fetch(recusalPath, {
+      method: 'PATCH',
+      headers: {
+        'Content-Type': 'application/json',
+        'X-CSRF-Token': document.querySelector('meta[name="csrf-token"]').content,
+        'Accept': 'application/json'
```

Comment:
Now I'm seeing 406 Not Acceptable error response from the server, but it works if I change this request header:
```suggestion
        'Accept': 'text/html'
```

<img width="803" alt="Screenshot 2025-02-05 at 2 35 55 PM" src="https://github.com/user-attachments/assets/73d6f76e-3442-466d-a573-b963a4185aee" />

Reply:
(the recusal still works without the header after it redirects, so this just reduces request errors on the server)